src/components/Search/Search.tsx

```diff
@@ -19,6 +21,8 @@ export const Search = (
 ): React.ReactElement => {
   const {
     onSubmit,
+    id = 'search-field',
```

Comment:
Lets name this `inputId` since `id` is an existing HTML attribute that could be captured in`...formProps` and passed down to the `<form>` also.

Reply:
Yep that occurred to me, thanks for the explanation.  Would `className` ever clash as well?

Reply:
since `className` is part of the destructure assignment, it won't be included in `formProps` so it should be fine. if it _wasn't_ destructured_ (and appended to the other classes using `classnames`) then yes, it could clash with the existing `className` prop, so it's important to handle that one specially :D